BUILDING.md

```diff
@@ -103,6 +103,7 @@ Prerequisites:
 * Basic Unix tools required for some tests,
   [Git for Windows](http://git-scm.com/download/win) includes Git Bash
   and tools which can be included in the global `PATH`.
+* vcbuild is provided by this repo as a batch file so if using powershell, use local syntax `./vcbuild nosign`
```

Comment:
I'd prefer this:

> Please note that `vcbuild` is a batch file provided with the Node.js source code. If using PowerShell, use local syntax. For example: `./vcbuild nosign`

Note that it should be a standalone sentence below the bulleted list of prerequisites, not an item on the list (as it is not a prerequisite).